src/Codeception/Coverage/Subscriber/LocalServer.php

```diff
@@ -129,10 +179,10 @@ public function afterSuite(SuiteEvent $e)
     /**
      * Allows Translating Remote Paths To Local (IE: When Using Docker)
      *
-     * @param \SebastianBergmann\CodeCoverage\CodeCoverage $coverage
-     * @return $this
+     * @param CodeCoverage $coverage
+     * @return LocalServer
```

Comment:
Are `@param` and `@return` annotations in docblock even necessary when type hints provide the same information?

Reply:
It depends. Tools like `robo` are able to infer the types and generate the documentation for parameters and return values ​​anyway? PHPStorm suggests adding them... but I don't think they are strictly necessary

Reply:
We don't generate documentation for this class.